test/test_common.py

```diff
@@ -43,7 +43,7 @@ def test_gloo_built(self):
         """Test that Gloo has been built if env is set."""
         gloo_rank = int(os.getenv('HOROVOD_RANK', -1))
         if gloo_rank >= 0:
-            self.assertEqual(gloo_built())
+            self.assertTrue(gloo_built())
```

Comment:
The old test is invalid:
`self.assertEqual` has 3 arguments so it will raise error, but launch gloo job will always return zero so the old test is always passed.